apps/asap-cli/package.json

```diff
@@ -7,6 +7,12 @@
     "node": "18.x"
   },
   "scripts": {
+    "algolia:delete-index": "ts-node --transpile-only src/cli.ts algolia:delete-index",
```

Comment:
isn't `TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}` necessary in some scripts like it was in the root package.json?

Reply:
No, we moved it into tsconfig in the TS_Node section for the entire app